packages/react-dom/src/events/__tests__/SyntheticWheelEvent-test.js

```diff
@@ -9,61 +9,106 @@
 
 'use strict';
 
-var SyntheticWheelEvent;
-
 describe('SyntheticWheelEvent', () => {
-  var createEvent;
+  var React;
+  var ReactDOM;
+  var createMouseEvent;
 
   beforeEach(() => {
-    // TODO: can we express this test with only public API?
-    SyntheticWheelEvent = require('../SyntheticWheelEvent').default;
-
-    createEvent = function(nativeEvent) {
-      var target = require('../getEventTarget').default(nativeEvent);
-      return SyntheticWheelEvent.getPooled({}, '', nativeEvent, target);
+    React = require('react');
+    ReactDOM = require('react-dom');
+    createMouseEvent = (type = '', options = {}) => {
+      const event = document.createEvent('MouseEvent');
+      event.initEvent(type, true, false);
+      return Object.assign(event, options);
     };
   });
 
   it('should normalize properties from the Event interface', () => {
-    var target = document.createElement('div');
-    var syntheticEvent = createEvent({srcElement: target});
+    const container = document.createElement('div');
+    const event = createMouseEvent('', {srcElement: container});
+    container.dispatchEvent(event);
 
-    expect(syntheticEvent.target).toBe(target);
-    expect(syntheticEvent.type).toBe(undefined);
+    expect(event.target).toBe(container);
+    expect(event.type).toBe('');
   });
 
   it('should normalize properties from the MouseEvent interface', () => {
-    expect(createEvent({which: 2, button: 1}).button).toBe(1);
+    const event = new MouseEvent('', {
+      bubbles: true,
+      cancelable: false,
+      which: 2,
+      button: 1,
+    });
+    expect(event.button).toBe(1);
   });
 
   it('should normalize properties from the WheelEvent interface', () => {
-    var standardEvent = createEvent({deltaX: 10, deltaY: -50});
-    expect(standardEvent.deltaX).toBe(10);
-    expect(standardEvent.deltaY).toBe(-50);
+    var events = [];
+    const container = document.createElement('div');
+    var onWheel = event => {
+      event.persist();
+      events.push(event);
+    };
+    const component = ReactDOM.render(<div onWheel={onWheel} />, container);
+    document.body.appendChild(container);
+
+    const node = ReactDOM.findDOMNode(component);
```

Comment:
`ReactDOM.render` already returns a DOM node. There is no need to call `findDOMNode` on a DOM node.